src/diffusers/models/transformers/transformer_flux.py

```diff
@@ -227,6 +227,7 @@ class FluxTransformer2DModel(
     _supports_gradient_checkpointing = True
     _no_split_modules = ["FluxTransformerBlock", "FluxSingleTransformerBlock"]
     _skip_layerwise_casting_patterns = ["pos_embed", "norm"]
+    _repeated_blocks = _no_split_modules
```

Comment:
can we just type the layer names out? when someone implement a new model based on flux, they may assume this two attribute are same here, (but they just happen to be the same in this case)